
{ ConnectionError: 用户 'sa' 登录失败。
    at Connection.tedious.once.err (/home/cairui/Node/myapp/node_modules/_mssql@4.0.4@mssql/lib/tedious.js:216:17)
    at Connection.g (events.js:292:16)
    at emitOne (events.js:96:13)
    at Connection.emit (events.js:188:7)
    at Connection.processLogin7Response (/home/cairui/Node/myapp/node_modules/_tedious@2.0.0@tedious/lib/connection.js:846:16)
    at Connection.message (/home/cairui/Node/myapp/node_modules/_tedious@2.0.0@tedious/lib/connection.js:1298:21)
    at Connection.dispatchEvent (/home/cairui/Node/myapp/node_modules/_tedious@2.0.0@tedious/lib/connection.js:687:45)
    at MessageIO.<anonymous> (/home/cairui/Node/myapp/node_modules/_tedious@2.0.0@tedious/lib/connection.js:602:18)
    at emitNone (events.js:86:13)
    at MessageIO.emit (events.js:185:7)
  code: 'ELOGIN',
  originalError:
   { ConnectionError: 用户 'sa' 登录失败。
       at ConnectionError (/home/cairui/Node/myapp/node_modules/_tedious@2.0.0@tedious/lib/errors.js:12:12)
       at Parser.<anonymous> (/home/cairui/Node/myapp/node_modules/_tedious@2.0.0@tedious/lib/connection.js:373:38)
       at emitOne (events.js:96:13)
       at Parser.emit (events.js:188:7)
       at Parser.<anonymous> (/home/cairui/Node/myapp/node_modules/_tedious@2.0.0@tedious/lib/token/token-stream-parser.js:54:15)
       at emitOne (events.js:96:13)
       at Parser.emit (events.js:188:7)
       at addChunk (/home/cairui/Node/myapp/node_modules/_readable-stream@2.3.3@readable-stream/lib/_stream_readable.js:284:12)
       at readableAddChunk (/home/cairui/Node/myapp/node_modules/_readable-stream@2.3.3@readable-stream/lib/_stream_readable.js:271:11)
       at Parser.Readable.push (/home/cairui/Node/myapp/node_modules/_readable-stream@2.3.3@readable-stream/lib/_stream_readable.js:238:10) message: '用户 \'sa\' 登录失败。', code: 'ELOGIN' },
  name: 'ConnectionError' }
Connection is closed.
{ status: '0', info: 'sql excute error', affectedRows: 0 }
   }

   code: 'ETIMEOUT',